frontend/pages/index.tsx

```diff
@@ -43,6 +46,7 @@ export default function Home() {
 				id="invitationTabHeaderId"
 				name="Invitations"
 				style="invitations"
+				notification={`${invitations?.groupInvitation?.length ?? ""}`}
```

Comment:
Does it work so 0 notifications won't show up?
Just looking at this, \``0 ?? ""`\` => `"0"` and `!!"0"` => `true` which would be displayed as badge content:

```ts
{notification && (
{"0" && ( // <- here
```

```suggestion
				notification={`${invitations?.groupInvitation?.length || ""}`}
```

Reply:
Yeah, it's working fine, because there is no 0. If there is no invitation the statement on the left is evaluated to false since `invitations` is an empty dictionary and `groupInvitation` is therefore undefined. That's why an empty string is passed to the component and will not be rendered. However, your suggestion is the more robust solution and I will change that.

Reply:
fixed in 70fc959e6dc847955b455327040c2507f9ada2fe

Reply:
got it, thanks for clarifying.